modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -0,0 +1,2166 @@
+----
+GRANT REMOVE PRIVILEGE ON DBMS TO privilegeRemover
+----
+
+The resulting role has privileges that only allow removing privileges.
+List all privileges for the role `privilegeRemover` as commands by using the following query:
+
+[source, cypher, role=noplay]
+----
+SHOW ROLE privilegeRemover PRIVILEGES AS COMMANDS
+----
+
+.Result
+[options="header,footer", width="100%", cols="m"]
+|===
+|command
+|"GRANT REMOVE PRIVILEGE ON DBMS TO `privilegeRemover`"
+a|Rows: 1
+|===
+
+The privileges to list, assign, and remove privileges can be granted via the `PRIVILEGE MANAGEMENT` privilege.
+See an example:
+
+[source, cypher, role=noplay]
+----
+GRANT PRIVILEGE MANAGEMENT ON DBMS TO privilegeManager
+----
+
+The resulting role has all privileges to manage privileges.
+List all privileges for the role `privilegeManager` as commands by using the following query:
+
+[source, cypher, role=noplay]
+----
+SHOW ROLE privilegeManager PRIVILEGES AS COMMANDS
+----
+
+.Result
+[options="header,footer", width="100%", cols="m"]
+|===
+|command
+|"GRANT PRIVILEGE MANAGEMENT ON DBMS TO `privilegeManager`"
+a|Rows: 1
+|===
+
+
+[[access-control-dbms-administration-execute]]
+== The DBMS `EXECUTE` privileges
+
+The DBMS privileges for procedure and user defined function execution can be assigned by using Cypher administrative commands.
+They can be granted, denied and revoked like other privileges.
+
+[NOTE]
+====
+For more details about the syntax descriptions, see xref:database-administration/syntax.adoc[].
+====
+
+.Execute privileges command syntax
+[options="header", width="100%", cols="3a,2"]
+|===
+| Command
+| Description
+
+| [source, syntax, role=noheader]
+GRANT [IMMUTABLE] EXECUTE PROCEDURE[S] name-globbing[, ...]
+  ON DBMS
+  TO role[, ...]
+| Enables the specified roles to execute the given procedures.
+
+| [source, syntax, role=noheader]
+GRANT [IMMUTABLE] EXECUTE BOOSTED PROCEDURE[S] name-globbing[, ...]
+  ON DBMS
+  TO role[, ...]
+| Enables the specified roles to execute the given procedures with elevated privileges.
+
+| [source, syntax, role=noheader]
+GRANT [IMMUTABLE] EXECUTE ADMIN[ISTRATOR] PROCEDURES
+  ON DBMS
+  TO role[, ...]
+| Enables the specified roles to execute procedures annotated with `@Admin`. The procedures are executed with elevated privileges.
+
+| [source, syntax, role=noheader]
+GRANT [IMMUTABLE] EXECUTE [USER [DEFINED]] FUNCTION[S] name-globbing[, ...]
+  ON DBMS
+  TO role[, ...]
+| Enables the specified roles to execute the given user defined functions.
+
+| [source, syntax, role=noheader]
+GRANT [IMMUTABLE] EXECUTE BOOSTED [USER [DEFINED]] FUNCTION[S] name-globbing[, ...]
+  ON DBMS
+  TO role[, ...]
+| Enables the specified roles to execute the given user defined functions with elevated privileges.
+|===
+
+The `EXECUTE BOOSTED` privileges replace the `dbms.security.procedures.default_allowed` and `dbms.security.procedures.roles` configuration parameters for procedures and user defined functions.
```

Comment:
Also missing updates from https://github.com/neo4j/docs-cypher/pull/733 (which I think you know since it has a comment to that effect)

Reply:
Yes, I'll transfer all the PRs after we merge this one.